ui/plugins/pluginpanel.html

```diff
@@ -206,6 +206,13 @@
           <label>Log parsing status:</label>
           <span>No logs</span>
         </li>
+        <li class="small" ng-click="backfillJob()">
+          <label>
+            <a title="Trigger this jobtype back to the previous time it was run">
+                Backfill this job
+            </a>
+          </label>
+        </li>
```

Comment:
@tojonmz thanks for your comment. :KWeirso commented ( https://bugzilla.mozilla.org/show_bug.cgi?id=1183923#c3 ) it to be not put in the "actions" bar since this will not be used as much. Thus I have shifted it to job details.